tests/cover/test_simple_characters.py

```diff
@@ -84,6 +84,23 @@ def test_find_something_rare():
         find(st, lambda c: unicodedata.category(c) != 'Zs')
 
 
+def test_whitelisted_characters_alone():
+    with pytest.raises(InvalidArgument):
+        characters(whitelist_characters=u'te02тест49st').example()
+
+
+def test_whitelisted_characters_override():
+    good_characters = u'te02тест49st'
+    st = characters(min_codepoint=ord('0'), max_codepoint=ord('9'),
+                    whitelist_characters=good_characters)
+
+    find(st, lambda c: c in good_characters)
```

Comment:
When there's no assertion about the value returned by `find`, they probably could.  (The exception is if there might be an exception in minimisation)

Fixing that is a big job though, so I've opened #734.